config/config.exs

```diff
@@ -1,3 +1 @@
 use Mix.Config
-
-config :forge_abi, decimal: 16
```

Comment:
in forge, update_config would override this with the value stored in forge state. In forge sdk, this config is useless since one should always retrieve from conn.